service/src/main/java/greencity/service/ubs/UBSManagementServiceImpl.java

```diff
@@ -658,20 +658,22 @@ private void collectEventAboutConfirmWaste(Map<Integer, Integer> confirmed, Orde
         for (Map.Entry<Integer, Integer> entry : confirmed.entrySet()) {
             Integer capacity = bagRepository.findCapacityById(entry.getKey());
             Optional<Bag> bagOptional = bagRepository.findById(entry.getKey());
-
             if (bagOptional.isPresent() && checkOrderStatusAboutConfirmWaste(order)) {
                 Optional<Long> confirmWasteWas = Optional.empty();
+                Optional<Long> startAmount = Optional.empty();
```

Comment:
I guess it is better to use `initial` instead of `start`

Reply:
I agree